test/jasmine/tests/shapes_test.js

```diff
@@ -75,33 +159,32 @@ describe('Test shapes:', function() {
 
     describe('Plotly.relayout', function() {
         it('should be able to add a shape', function(done) {
-            var pathCount = countShapePaths();
+            var pathCount = countShapePathsInUpperLayer();
             var index = countShapes(gd);
             var shape = getRandomShape();
 
-            Plotly.relayout(gd, 'shapes[' + index + ']', shape).then(function() {
-                expect(countShapeLayers()).toEqual(1);
-                expect(countShapePaths()).toEqual(pathCount + 1);
+            Plotly.relayout(gd, 'shapes[' + index + ']', shape).then(function()
+            {
```

Comment:
I'm trying to keep lines 80 chars or less. Shall I break the line before the function, like this:

```
            Plotly.relayout(gd, 'shapes[' + index + ']', shape).then(
                function() {
                    expect(countShapePathsInUpperLayer())
                        .toEqual(pathCount + 1);
                    expect(getLastShape(gd)).toEqual(shape);
                    expect(countShapes(gd)).toEqual(index + 1);
                }).then(done);
```